components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa.Edge.Publisher/tests/Engine/pn_assets.json

```diff
@@ -11,7 +11,9 @@
     "OpcNodes": [
       {
         "Id": "nsu=http://microsoft.com/Opc/OpcPlc/;s=StepUp",
-        "DataSetFieldId": "nsu=http://microsoft.com/Opc/OpcPlc/;s=StepUp"
+        "DataSetFieldId": "nsu=http://microsoft.com/Opc/OpcPlc/;s=StepUp",
```

Comment:
Same comment as previously. Let's keep the version of the file without `DataSetFieldId`, `OpcSamplingInterval` and `OpcPublishingInterval` properties and have a new file with those to make sure OPC Publisher would recognize both as having valid schemas.

Reply:
Assuming you meant to say "keep DataSetFieldId" in the above comment.

Reply:
Adding InlineData in test to check pn_assets.json with previous and new versions both, presuming that should cover what we need?

Reply:
InlineData data changes will be pushed in next commit. Please feel free to suggest otherwise.

Reply:
Thanks.